morpheus/_lib/src/objects/tensor_object.cpp

```diff
@@ -107,4 +85,26 @@ const std::vector<TensorIndex>& TensorView::stride() const
 {
     return m_stride;
 }
+
```

Comment:
Why comment? Can we delete?

Reply:
These files were copied verbatim from Neo so I was trying to minimize changes in them. Will remove